focus-ios/focus-ios-tests/XCUITest/CopyPasteTest.swift

```diff
@@ -20,7 +20,7 @@ class CopyPasteTest: BaseTestCase {
         loadWebPage("mozilla.org")
         waitForWebPageLoad()
         urlBarTextField.press(forDuration: 2)
-        waitForExistence(app.collectionViews.menuItems.firstMatch)
+        waitForExistence(app.menuItems.firstMatch)
```

Comment:
iOS 15 and 16 do not include the menu in a `collectionViews`. This change should work for iOS 15, 16 and 17.